administrator-manual/en/webtop5.rst

```diff
@@ -291,15 +291,103 @@ In the same way, you can always share your contacts by selecting the directory y
 Select the recipient user of the share (or Group), and enable permissions for both the folder and the individual items.
 
 
-Mail tags
-=========
+Custom cross labels
+==================
 
-You can tag each message with different colored labels.
-Just select a message, right-click and select :guilabel:`Tag`.
+There are two types of labels:
 
-You can edit existing tags or add new ones selecting :guilabel:`Manage tags`.
+- **Private**: not usable for custom fields and not visible to other users
+- **Shared**: usable for custom field panels and visible to other users
+
+In order to manage the "Shared" labels it is necessary to activate a specific authorization via the admin panel:
```

Comment:
First, specify that all users can use private labels.
Then, introduce the required authorization.
Finally, remove the node at line 308.